spring-mvc-simple-2/src/main/java/com/baeldung/exclude_urls_filter/filter/LogFilter.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.exclude_urls_filter.filter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.annotation.Order;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+//@Component
+@Order(1)
+public class LogFilter implements Filter {
+
+    private final Logger logger = LoggerFactory.getLogger(LogFilter.class);
+
+    @Override
+    public void doFilter(
+            ServletRequest request,
+            ServletResponse response,
+            FilterChain chain) throws IOException, ServletException {
+
+        String path = ((HttpServletRequest) request).getRequestURI();
+        String contentType = request.getContentType();
+        logger.info("Path logged:" + path);
+        if (!path.endsWith("health")) {
+            logger.info("Mime type of request body is:" + contentType);
```

Comment:
let's use the logging features .ingo("... {}", var1, ...)

Reply:
Done 🙂